src/Runtime/include/OnnxMlir.h

```diff
@@ -0,0 +1,289 @@
+//===-------- RtMemRef.h - external RtMemRef C/C++ API call header --------===//
+//
+// Copyright 2019-2020 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file contains declaration of external RtMemRef data structures and
+// helper functions.
+//
+//===----------------------------------------------------------------------===//
+#ifndef __RTMEMREF_H__
+#define __RTMEMREF_H__
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdbool.h>
+#include <stdint.h>
+#endif
+
+/*! \mainpage ONNX-MLIR Model Execution API documentation
+ *
+ * \section intro_sec Introduction
+ *
+ * ONNX-MLIR project comes with an executable `onnx-mlir` capable
+ * of compiling onnx models to a shared library. In this documentation, we
+ * explain and document ways to interact programmatically with the compiled
+ * shared library using the API's defined in `src/Runtime/include/OnnxMlir.h`.
+ *
+ * \section Important Data Structures
+ * \subsection RtMemRef
```

Comment:
in the list of functions operating on RtMemRef, I did not find one that accesses the data

rmrGetElement(rmr, indices)
rmrSetElement(rmr, indices, value)

Reply:
Probably a thing to add next, if there's a use case for them.